cmd/add.go

```diff
@@ -99,5 +104,5 @@ func init() {
 
 	addFromFileCmd.AddCommand(addDomainsCmd)
 
-	addDomainsCmd.Flags().String("ip", "127.0.0.1", "domains ip")
+	addDomainsCmd.Flags().String("ip", "", "domains ip")
```

Comment:
Not sure I understand this change. The previous line behaves exactly as this, but without the if statement needed. The second parameter to `Flags().String()` is the default value. 

Can you explain why this is needed?

Reply:
Sorry, I was testing how to add an ip and did not realize after changing the code that it was already done like that :|.

I guess only the documentation is necessary.